Sources/Core/API/TextureManager.swift

```diff
@@ -7,6 +7,12 @@
 import Foundation
 import CoreGraphics
 
+public enum ErrorMode {
+  case IgnoreError
+  case LogError
+  case TriggerAssertOnError
```

Comment:
I don't think it's neccessary to add the `Error` suffix to all cases, I think they can simply be: 
```
ignore
log
assert
```

Reply:
Great tip. Fixed. 👍